geometry_msgs/Polygon area
---
string[] names
geometry_msgs/Point[] locations
